README.md

```diff
@@ -127,16 +127,18 @@ global.instancename = dev-from-ide
 
 # DE4A Kafka settings
 de4a.kafka.enabled=false
+# Enables Kafka connection via HTTP (it must be used when HTTP connections are being established via proxy)
+de4a.kafka.http.enabled=false
 # Kafka server address (default: de4a-dev-kafka.egovlab.eu:9092)
 de4a.kafka.url=de4a-dev-kafka.egovlab.eu:9092
-# Add any suffix if you have multiple Connector instances on the same kafka server (default: DE4A-CONNECTOR)
-de4a.kafka.topic=DE4A-CONNECTOR
+# Add any suffix if you have multiple Connector instances on the same kafka server (default: de4a-connector)
+de4a.kafka.topic=de4a-connector
```

Comment:
Maybe we should establish a naming strategy here. ie 
de4a-<country-code>-<partner-name>
which would mean every partner would be easily distinguishable in the package tracker.
Egovlab would be de4a-se-egovlab, spain would be de4a-es-sgad (or some such)

Reply:
yes, good point. Totally agree.